ApnsPHP/SharedConfig.php

```diff
@@ -127,7 +127,7 @@ abstract class SharedConfig
      *         with key (Bundled PEM).
      * @param  $protocol @type integer Protocol.
      */
-    public function __construct($environment, $providerCertificateFile, $protocol = self::PROTOCOL_BINARY)
+    public function __construct(int $environment, string $providerCertificateFile, int $protocol = self::PROTOCOL_BINARY)
```

Comment:
```suggestion
    public function __construct(int $environment, string $providerCertFile, int $protocol = self::PROTOCOL_BINARY)
```

Reply:
Yeah, I saw. But I'd rather tackle this together with other style changes